pallets/dca/src/lib.rs

```diff
@@ -199,17 +191,27 @@ pub mod pallet {
 						}
 					}
 				}
+				match schedule.order {
+					Order::Buy { route, .. } => {
+						weight.saturating_accrue(T::AmmTradeWeights::buy_and_calculate_buy_trade_amounts_weight(&route))
+					}
+					Order::Sell { route, .. } => weight
+						.saturating_accrue(T::AmmTradeWeights::sell_and_calculate_sell_trade_amounts_weight(&route)),
+				}
 			}
 
 			weight
```

Comment:
(I comment here as I can not comment unchanged file in the review)

I expected that the [get_trade_weight()](https://github.com/galacticcouncil/HydraDX-node/blob/1de25855e854abe5ba596a1f4b72a0aba5da839f/pallets/dca/src/lib.rs#L1036) function will also change, as it is used to calculate the fee we deduct from the user at the beginning of each trade. 

Or why was it not changed?

Reply:
We don't need to change it because it now benchmarks only DCA code. The router and AMMs are benchmarked separately.